### DISCUSSION TITLE
0.18.0-beta-2

### DISCUSSION THREAD

**Discussion by redphx** [redphx/better-xcloud-android#129] [Announcements]
<img width="476" alt="Screenshot" src="https://github.com/user-attachments/assets/b2c05e86-ea53-4148-a599-9c3995125240">
<img width="476" alt="Screenshot" src="https://github.com/user-attachments/assets/979b6908-68f2-4ce2-b310-03fe6b76cdf6">

## 0.18.0-beta-2 changelogs

> [!NOTE]
> After installing, go to App settings > Better xCloud script variant > Beta to use the beta version of the script

### Added
- Add WebView Manager dialog
    - Allow changing WebView implementation and download new one
    - WebView will be downloaded in the background. Check Notification for the download progress
- Bypass "Update FireOS" screen on Fire TV Stick

### Changed
- Always force low-latency audio (and hide this settings in App settings)

### Fixed
- Fix not refresh the WebView after changing App settings

### Added
- Add `Optimize WebView performance` feature.
    - Default is Off
    - Improve WebView performance on low-specs devices, especially Android TVs or Fire TV Sticks
    - This will automatically apply all the neccessary flags in [this guide](https://better-xcloud.github.io/guide/android-webview-tweaks/). If you previously enabled/disabled these flags, you can set them back to `Default`.
    - It's normal if you see a `WebView DevTools flags` notification when using this feature. You can simply turn it off in the System settings.
    - It's recommended to use with Thorium WebView for the best performance.

### Changed
- Enable `Force low-latency audio` feature by default
- Update the UI of WebView selection dialog
- Re-enable the "Log in remotely" feature

### Fixed
- Fix compatibiliy issues with Fire TV Sticks

### What to do if you found a bug
- Report on [Telegram channel](https://t.me/betterxcloudchat) or in [this discussion](https://github.com/redphx/better-xcloud-android/discussions/128) with these info:
    - Device info
    - WebView info
    - Screenshot/video
    - Logcat
- You can re-install the stable version if you want without having to uninstall the beta version

<hr /><em>This discussion was created from the release <a href='https://github.com/redphx/better-xcloud-android/releases/tag/a0.18.0-beta-2'>0.18.0-beta-2</a>.</em>

**Comment by KingNothing81** [comment]
Hi …

I can confirm that both issues reported here …
https://github.com/redphx/better-xcloud-android/discussions/128#discussioncomment-10225704
… are fixed for me running the current beta versions of app and script.

Thanks for fixing.
